ip-visit-sqs-consumer/mirrord.json

```
@@ -0,0 +1,13 @@
+{
+    "target": "deployment/ip-visit-sqs-consumer",
+    "feature": {
+        "split_queues": {
+            "ip-count": {
+                "queue_type": "SQS",
+                "message_filter": {
+                    "x-pg-tenant": "Avi.+"
```

Nitpick: the regex is matched anywhere within the value, so the `+` is not necessary, and including it could make people think the complete attribute value must match the pattern for the message to be matched.

```suggestion
                    "x-pg-tenant": "Avi."
```